juniper_official/Solutions/Traffic-Blackhole/lookup-interrupts.rule

```diff
@@ -0,0 +1,73 @@
+/*
+	* REQ: REQ-222
+	* Rule to collect Lookup Block CM Errors and Interrupts for Blackhole Detection RCA.
+*/
+healthbot {
+	topic hardware.fpc {
+		rule lookup-interrupts {
+			keys [ fpc-name error-name ];
+			synopsis "Lookup Block CM Errors and Interrupts";
+			description "Lookup Block CM Errors and Interrupts";
+			sensor sensor-interrupts {
+				synopsis "Lookup Block CM Errors and Interrupts";
+				description "Lookup Block CM Errors and Interrupts";
+				open-config {
+					sensor-name /components/component/;
+					frequency 60s;
+				}
+			}
+			field fpc-name {
```

Comment:
For keeping common names, can we rename this to "fpc"? Keys section also should be adjusted accordingly

Reply:
Changed to fpc